src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,60 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+
 public class WorkWithFile {
+    public static final String DELIMITER = ",";
+    public static final String OPERATION = "buy";
+    private int supplyAmount = 0;
+    private int buyAmount = 0;
+    private final StringBuilder stringBuilder = new StringBuilder();
+
     public void getStatistic(String fromFileName, String toFileName) {
+        File fromFile = new File(fromFileName);
+        final File toFile = new File(toFileName);
+
+        readDataFromTheFile(fromFile);
+
+        int result = supplyAmount - buyAmount;
+
+        stringBuilder.setLength(0);
+        stringBuilder.append("supply,").append(supplyAmount).append(System.lineSeparator())
+                .append("buy,").append(buyAmount).append(System.lineSeparator())
+                .append("result,").append(result);
```

Comment:
Using StringBuilder to concatenate data in a loop is good, but here it's not necessary since there's no loop. Directly writing the result string to the file would be more efficient.